Doc/https and credentials

jmunoznaranjo opened the pull request:
This PR adds much needed documentation about the configuration of admin credentials (HTTPS was already there).

rahulreddy commented:
👍 We need this!

LaurenSpiegel commented on README.md:
```diff
@@ -93,28 +93,35 @@ $ bin/vaultclient create-account --name account0 --email d3v@null \
+
+3. Pass the filepath in the command line with option `--config`
+
+An example of the latter is shown next:
```
An example of the third option is:

jmunoznaranjo commented:
@rahulreddy Indeed, I opened against master by mistake  :weary:

Can you guys check the new one https://github.com/scality/vaultclient/pull/82 ? @LaurenSpiegel I've addressed your comment there.
